hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -178,6 +191,52 @@
       act:     "req"
       desc:    "This is the debug request interrupt going to the main processor."
     },
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_escalate_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg",
+      desc:    '''
+               Escalation enable signal coming from life cycle controller, used for invalidating
+               the latched lc_hw_debug_en state inside the strap sampling logic.
+               ''',}
```

Comment:
Nit: I realise that it has no effect, but I'm not a big fan of the ",}" syntax. Maybe drop the comma?

Reply:
Sorry to be annoying but I think the comma is still here on the first item (but thanks for dropping it on the later ones!)

Reply:
finally fixed.